Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -64,6 +65,114 @@ To confirm the object store is configured, wait for the RGW pod(s) to start:
 kubectl -n rook-ceph get pod -l app=rook-ceph-rgw
 ```
 
+To consume the object store, continue below in the section to [Create a bucket](#create-a-bucket).
+
+### Create Local Object Store(s) with Shared Pools
+
+The below sample will create one or more object stores. Shared Ceph pools will be created, which reduces the overhead of additional Ceph pools for each additional object store.
+
+Data isolation is enforced between the object stores with the use of Ceph RADOS namespaces. The separate RADOS namespaces do not allow access of the data across object stores.
+
+!!! note
+    This sample requires *at least 3 OSDs*, with each OSD located on a *different node*.
+
+The OSDs must be located on different nodes, because the [`failureDomain`](../../CRDs/Block-Storage/ceph-block-pool-crd.md#spec) is set to `host` and the `erasureCoded` chunk settings require at least 3 different OSDs (2 `dataChunks` + 1 `codingChunks`).
+
+#### Shared Pools
+
+Create the shared pools that will be used by each of the object stores.
+
+```yaml
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
+metadata:
+  name: rgw-root
+  namespace: rook-ceph # namespace:cluster
+spec:
+  name: .rgw.root
```

Comment:
If `.rgw.root` already exists (or any other pool when its CephBlockPool when the CR is created), Rook will attempt to reconcile it according to the CR. If the pool settings are different, Rook will update them. If the pool already exists and the spec isn't different from the Ceph pool config, then it will be a no-op.